### PR TITLE
docs: Small edits for spelling, grammar, formatting.

### PR DESCRIPTION
**Pull request by Zearin**
This is a redo of #711 :)

### CODE REVIEW

File: docs/_includes/content.md [hunk 1]
```diff
@@ -3358,10 +3357,10 @@ unstable_features = ["CTRL_C_HANDLING"]
 
 Below is a list of currently existing unstable features
 
-* CTRL_C_HANDLING - Adds ctrl-c handler which will stop any currently running command invoked by the current task and exit cargo-make
+* **CTRL_C_HANDLING** - Adds <kbd>ctrl-c</kbd> handler, which will stop any currently running command invoked by the current task and exit cargo-make
 
 <a name="usage-cli"></a>
-### Cli Options
+### CLI Options
```
Comment (sagiegurari) [inline docs/_includes/content.md:3363]: can you also update the nav.md for this?

**Review by sagiegurari — CHANGES_REQUESTED** [review]

**Comment by sagiegurari** [comment]
@Zearin thanks a lot for this huge effort. i'm bound to ruin it over time :D
anyhow, just got one small fix needed and than (or then?) i'll merge it.

File: docs/_includes/content.md (hunk 1 above)
Comment (Zearin) [reply docs/_includes/content.md:3363]: Sure!

**Review by sagiegurari — APPROVED** [review]

**Comment by sagiegurari** [comment]
@Zearin thanks a lot. merging

**Pull request merged**